unitTests/ui/ResultListTest.ts

```diff
@@ -801,60 +800,38 @@ export function ResultListTest() {
 
           beforeEach(() => {
             jasmine.clock().install();
-            infiniteScrollContainer = document.createElement('div');
-            scrollToSpy = spyOn(infiniteScrollContainer, 'scrollTo');
-
-            // Inner value of scrollTop.
-            let innerValue = 500;
-            scrollTopSpy = jasmine.createSpy('scrollTop', v => (innerValue = v));
-            Object.defineProperty(infiniteScrollContainer, 'scrollTop', {
-              get: () => innerValue,
-              set: scrollTopSpy
-            });
+            scrollToTopSpy = spyOn(ResultListUtils, 'scrollToTop');
+
             allResult = FakeResults.createFakeResults(50);
           });
 
           afterEach(() => {
             jasmine.clock().uninstall();
           });
 
-          it('true, should set the value ScrollTop of the HTMLElement to 0 in IE11', done => {
-            infiniteScrollContainer.scrollTo = undefined;
-            test = setupResultList();
-            test.cmp.displayMoreResults(50).then(() => {
-              Simulate.query(test.env);
-              jasmine.clock().tick(0);
-
-              expect(scrollTopSpy).toHaveBeenCalledWith(0);
-
-              done();
-            });
-          });
-
-          it('true, should call the ScrollTo method of the HTMLElement', done => {
+          it('true, should call the scrollToTop of the ResultListUtils', done => {
             test = setupResultList();
             test.cmp.displayMoreResults(50).then(() => {
               Simulate.query(test.env);
               jasmine.clock().tick(0);
 
-              expect(scrollToSpy).toHaveBeenCalledWith(0, window.pageYOffset + test.cmp.element.getBoundingClientRect().top);
+              expect(scrollToTopSpy).toHaveBeenCalledWith(test.cmp.root);
 
               done();
             });
           });
 
-          it('false, should not set the value ScrollTop of the HTMLElement and should not call the ScrollTo method of the HTMLElement', done => {
+          it('false, should not call the scrollToTop of the ResultListUtils', done => {
             test = setupResultList({
               enableInfiniteScroll: true,
-              infiniteScrollContainer: infiniteScrollContainer,
+              infiniteScrollContainer: undefined,
```

Comment:
Hmm, maybe related to the comment above, when the container is `undefined`, is there an error in the console?